Jv rox 25727 show external entity ip information in the network graph (#84248)
openshift/openshift-docs: modules/querying-external-entities-using-api.adoc

// Module included in the following assemblies:
//
// * operating/visualizing-external-entities.adoc
:_mod-docs-content-type: REFERENCE
[id="querying-external-entities-using-api_{context}"]
= Querying external IP addresses by using the API

[role="_abstract"]

> You can get the information about external IP addresses associated with a specific cluster by using the following endpoints:

Reviewer: ```suggestion
You can get information about the external IP addresses associated with a specific cluster by using the following endpoints:
```

> * `/v1/networkgraph/cluster/{clusterId}/externalentities`: It returns a list of external entities for a given cluster ID. Each entity includes the following information:

Author: Committing the first option.

// Module included in the following assemblies:
//
// * operating/visualizing-external-entities.adoc
:_module-type: REFERENCE
[id="querying-external-entities-using-api_{context}"]
= Querying external IP addresses by using the API

[role="_abstract"]
You can get the information about external IP addresses associated with a specific cluster by using the following endpoints:

* `/v1/networkgraph/cluster/{clusterId}/externalentities`: It returns a list of external entities for a given cluster ID. Each entity includes the following information:
+
** *Name*: The name of the external entity.
** *CIDR block*: The CIDR block associated with the entity.

> ** *Default entity*: Indicates whether the entity is considered a default entity.

Author: ```suggestion
** *Default entity*: Indicates that the entity is a CIDR-block definition provided by the system.
```

** *Discovered*: Indicates whether the entity was discovered. The value can be:

> *** `discovered` means that the external IP address does not match any user-specified CIDR block.
> *** `default` means that the external IP address matches one of the system-specified CIDR blocks.

Author: ```suggestion
```
Discovered can be true or false.
